en/operations/configuration-server.html

```diff
@@ -206,64 +206,57 @@ <h2 id="start-sequence">Start sequence</h2>
 
 <h2 id="scaling-up">Scaling Up</h2>
 <p>
-  Add config server nodes for increased fault tolerance. Procedure:
+  Add config server nodes for increased fault tolerance.
+  Read up on <a href="#zookeeper-configuration">ZooKeeper configuration</a> before continuing.
+  Although it is <em>possible</em> to add more than one config server at a time,
+  doing it one by one is recommended, to keep the ZooKeeper quorum intact.
+</p>
+<p>
+  Due to the ZooKeeper majority vote, use one or three config servers.
+</p>
+<p>
+  To <em>replace</em> config servers, the easiest way is to remove a server and add the new one with the same name -
+  updating <em>VESPA_CONFIGSERVERS</em> is then not needed, and no nodes need restarting.
```

Comment:
I am unsure if what we are saying about no nodes needing restart is correct. Clients will need to lookup IP from hostnames often/regularly for this to work. Then again, I think we do this in hosted Vespa, so...

Reply:
right. so not sure what do put here

Reply:
I'll pick this up again over the weekend and discuss this with some others to see what makes sense